test/nodes/test_prompt_node.py

```diff
@@ -797,6 +797,37 @@ def test_complex_pipeline_with_multiple_same_prompt_node_components_yaml(tmp_pat
     assert pipeline is not None
 
 
+class TestTokenLimit:
+    @pytest.mark.integration
+    def test_hf_token_limit_warning(self, prompt_node, caplog):
+        tt = PromptTemplate(
+            name="too-long-temp",
+            prompt_text="Repeating text" * 200 + "Docs: $documents; Answer:",
+            prompt_params=["documents"],
+        )
+        with caplog.at_level(logging.WARNING):
+            _ = prompt_node.prompt(tt, documents=["Berlin is an amazing city."])
+            assert "The prompt has been truncated from 812 tokens to 412 tokens" in caplog.text
+            assert "and answer length (100 tokens) will fit within the max token limit (512 tokens)." in caplog.text
```

Comment:
`will fit` needs to be changed to `fits` to match the log message that is created.